packages/fetchai/connections/http_server/readme.md

```diff
@@ -0,0 +1,5 @@
+# HTTP server connection
+This connection wraps a http server. It consumes requests from clients, translates them into messages for the AEA, waits for a response message from the AEA, then serves the response to the client.
+
+## Usage
+First, add the connection to your AEA project (`aea add connection fetchai/http_server:0.5.0`). Then, update the `config` in `connection.yaml` by providing a `host` and `port` of the server. Optionally, provide a path to an OpenApi spec (https://swagger.io/docs/specification/about/) for request validation.
```

Comment:
Please replace "OpenApi" with "OpenAPI".
Also, consider using the Markdown syntax for links: [OpenAPI](https://swagger.io/docs/specification/about/)